File: gabriel/Problema0280/Problema0280.py
vetorA = [0] * 10

for i in range(10):
  vetorA[i] = int(input("Digite o valor da posição "+str(i)+ ": "))

vetorB = [0] * 10

j = int(0)

for i in range(10):
  if(vetorA[i] > 0):
    vetorB[j] = vetorA[i]
    j = j + 1

for i in range(j):
    print(vetorB[i])

File: gabriel/Problema0280/Problema0280.java
package Problema0280;

public class Problema0280 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
    
        int [] vetorA = new int[10];
        int [] vetorB = new int[10];

        System.out.println("Preencha o vetor A");
        for(int i=0; i<10; i++){
            System.out.println("Digite o valor inteiro da posiçao " +i);
            vetorA[i] = Integer.valueOf(input.nextLine());
        }
        int j=0;
        for(int i=0; i<10; i++){
            if(vetorA[i] > 0){
                vetorB[j] = vetorA[i];
                j++;
            }
        }
        System.out.println("Vetor B");
        for(int i=0; i<j; i++){
            System.out.println(vetorB[i]);
        }
    }
}
    
   
